zproject/urls.py

```diff
@@ -144,6 +144,7 @@
     url(r'^api/v1/fetch_google_client_id$', 'api_fetch_google_client_id'),
 
     # Sorted integration-specific webhook callbacks.
+    url(r'^api/v1/external/airbrake',       'webhooks.airbrake.api_airbrake_webhook'),
```

Comment:
This is missing a `$` at the end of the pattern; can you fix that here (and maybe doing another PR to fix the missing `$`s in the below).  I'll fix the ones merged so far; you should update your open PRs in addition to fixing this here.